generators/entity/index.js

```diff
@@ -428,8 +428,8 @@ module.exports = EntityGenerator.extend({
             this.entityPluralFileName = entityNamePluralizedAndSpinalCased + this.entityAngularJSSuffix;
             this.entityServiceFileName = entityNameSpinalCased + this.entityAngularJSSuffix;
             this.entityAngularName = this.entityClass + _.upperFirst(_.camelCase(this.entityAngularJSSuffix));
-            this.entityStateName = entityNameSpinalCased + this.entityAngularJSSuffix;
-            this.entityUrl = entityNameSpinalCased + this.entityAngularJSSuffix;
+            this.entityStateName = _.kebabCase(this.entityAngularName);
```

Comment:
its slightly different, if we have angularSuffix="MySuffix" and an entity "MyEntity", our angularName would be "MyEntityMySuffix" and our entityStateName would be "my-entity-my-suffix" where previously it was "my-entity-MySuffix", i think.